### PR TITLE
add Link.GetRoutes() method

### PR DESCRIPTION
**Pull request by nustiueudinastea**
This PR adds a GetRoutes() method to the Link interface and to the Linux implementation, which can be used to retrieve all routes (IPv4 and IPv6) for a given interface.

Dependencies are also updated. Updating to the latest github.com/jsimonetti/rtnetlink fixed a bug that prevents retrieving all routes for an interface.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ cmd/wboxd/wboxd
 cmd/wboxd/wboxd.toml
 cmd/wbox/wbox
 cmd/wbox/wbox.toml
+vendor/
```
Comment (foxcpp) [inline .gitignore]: Using `.git/info/exclude` for system/editor-specific is generally a good idea.
Could you remove these?

File: linkmgr/rtnetlink_linux.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"net"
 	"strconv"
 
+	"cuelang.org/go/pkg/strings"
```
Comment (foxcpp) [inline linkmgr/rtnetlink_linux.go]: Use standard library import path `"string"`.

**Review by foxcpp — CHANGES_REQUESTED** [review]

**Comment by foxcpp** [comment]
While I appreciate your contribution, note that `linkmgr` package in wirebox is not meant for use as a library - there are no API stability guarantees and I am not willing to make any.

File: linkmgr/rtnetlink_linux.go (hunk 2 above)
Comment (nustiueudinastea) [reply linkmgr/rtnetlink_linux.go]: Most definitely, auto import tripped me

File: .gitignore (hunk 1 above)
Comment (nustiueudinastea) [reply .gitignore]: Sure!

**Comment by nustiueudinastea** [comment]
Ok got it. I don't mind breaking changes, this is a very young project. I have a MacOS implementation for the linkmgr package, which I planned to submit as a PR. If you prefer not to have these contributions let me know and I can keep them in my repo. Cheers!

**Comment by foxcpp** [comment]
>I have a MacOS implementation for the linkmgr package, which I planned to submit as a PR. 

That would be amazing!

**Review by foxcpp — APPROVED** [review]
LGTM

**Comment by foxcpp** [comment]
Thanks!

**Comment by nustiueudinastea** [comment]
Thank you for this nice package!

**Pull request merged**